crates/viewer/re_component_ui/src/entity_path.rs

```diff
@@ -0,0 +1,27 @@
+use re_types::components::EntityPath;
+use re_viewer_context::{MaybeMutRef, ViewerContext};
+
+pub(crate) fn edit_or_view_entity_path(
+    ctx: &ViewerContext<'_>,
+    ui: &mut egui::Ui,
+    path: &mut MaybeMutRef<'_, EntityPath>,
+) -> egui::Response {
+    if let Some(path) = path.as_mut() {
+        // A suggestion mechanism similar to the one in `space_view_space_origin_widget_ui` would be nice.
+        let mut string = path.to_string();
+        let response = ui.text_edit_singleline(&mut string);
+        *path = string.into();
```

Comment:
Have you tried this? This will live-edit the path on each keyboard press, which may be a bit crazy, but should be fine.

Reply:
I played around with it, yes :). But the caller of this actually checks for `changed` response and will only write back to the blueprint store if that is set. 
That's done because checking for actual changes is hard and not what we want: If a fallback gets edited and ends up being the fallback, that's still an override happening